thirdparty/runtime_environments.go

```diff
@@ -38,8 +39,8 @@ func NewRuntimeEnvironmentsClient(baseURL string, apiKey string) *RuntimeEnviron
 	return &c
 }
 
-// getImageNames returns a list of strings containing the names of all images from the runtime environments API.
-func (c *RuntimeEnvironmentsClient) getImageNames(ctx context.Context) ([]string, error) {
+// GetImageNames returns a list of strings containing the names of all images from the runtime environments API.
+func (c *RuntimeEnvironmentsClient) GetImageNames(ctx context.Context) ([]string, error) {
```

Comment:
One thing I just realized is it would be nice if these came sorted in alphabetical order. I think you can use `sort.Strings` to do that